requirements/base.txt

```diff
@@ -7,3 +7,4 @@ tensorboard>=1.14
 future>=0.17.1  # required for builtins in setup.py
 # pyyaml>=3.13
 PyYAML>=5.1  # OmegaConf requirement
+torchtext>=0.3.1
```

Comment:
Torchtext as mandatory dependency? When we refused torchvision.... @williamFalcon

Reply:
agree, should go to extras. I was not fast enough to properly review :(

Reply:
fixing in #2283

Reply:
sorry, didn't catch that. i thought it was in extras